### ISSUE TITLE
Add random state

### ISSUE THREAD

**Issue by rasbt** [QuantumChamploo/RaschkaResearch#1]
https://github.com/QuantumChamploo/RaschkaResearch/blob/5c3e36c30b22eaf07721a25cf9cf52069ba26842/svm.py#L23

I would specify the random seed here for reproducibility, e.g.,

    X_train, X_test, Y_train, Y_test = train_test_split(train_x, train_y, test_size=0.6, shuffle=True, random_state=123)
